scripts/kconfig/kconfigfunctions.py

```diff
@@ -300,9 +329,13 @@ def dt_compat_enabled(kconf, _, compat):
         "dt_str_val": (dt_str_val, 1, 1),
         "dt_compat_enabled": (dt_compat_enabled, 1, 1),
         "dt_chosen_label": (dt_chosen_label, 1, 1),
-        "dt_chosen_reg_addr": (dt_chosen_reg_addr, 1, 3),
-        "dt_chosen_reg_size": (dt_chosen_reg_size, 1, 3),
-        "dt_node_reg_addr": (dt_node_reg_addr, 1, 3),
-        "dt_node_reg_size": (dt_node_reg_size, 1, 3),
+        "dt_chosen_reg_addr_int": (dt_chosen_reg, 1, 4),
+        "dt_chosen_reg_size_int": (dt_chosen_reg, 1, 4),
+        "dt_chosen_reg_addr_hex": (dt_chosen_reg, 1, 4),
+        "dt_chosen_reg_size_hex": (dt_chosen_reg, 1, 4),
+        "dt_node_reg_addr_int": (dt_node_reg, 1, 4),
+        "dt_node_reg_size_int": (dt_node_reg, 1, 4),
+        "dt_node_reg_addr_hex": (dt_node_reg, 1, 4),
+        "dt_node_reg_size_hex": (dt_node_reg, 1, 4),
```

Comment:
Maybe slightly easier to read in this order:

```python
"dt_chosen_reg_addr_int": (dt_chosen_reg, 1, 4),
"dt_chosen_reg_addr_hex": (dt_chosen_reg, 1, 4),
"dt_chosen_reg_size_int": (dt_chosen_reg, 1, 4),
"dt_chosen_reg_size_hex": (dt_chosen_reg, 1, 4),
"dt_node_reg_addr_int": (dt_node_reg, 1, 4),
"dt_node_reg_addr_hex": (dt_node_reg, 1, 4),
"dt_node_reg_size_int": (dt_node_reg, 1, 4),
"dt_node_reg_size_hex": (dt_node_reg, 1, 4),
```

Reply:
done